packages/clients/textLocator/src/plugins/GeometrySearch/store/actions/setupTooltip.ts

```diff
@@ -0,0 +1,50 @@
+import { Tooltip, getTooltip } from '@polar/lib-tooltip'
+import { Feature, Overlay } from 'ol'
+import { vectorLayer } from '../../utils/vectorDisplay'
+import { getPrimaryName } from '../../../../utils/coastalGazetteer/getPrimaryName'
+
+const localeKeys: [string, string][] = [
+  ['h2', 'plugins.geometrySearch.tooltip.title'],
+]
+
+export function setupTooltip({ rootGetters: { map } }) {
+  let element: Tooltip['element'], unregister: Tooltip['unregister']
+  const overlay = new Overlay({
+    positioning: 'bottom-center',
+    offset: [0, -5],
+  })
+  map.addOverlay(overlay)
+  map.on('pointermove', ({ pixel, dragging, originalEvent }) => {
+    if (dragging || ['touch', 'pen'].includes(originalEvent.pointerType)) {
+      return
+    }
+    let hasFeatureAtPixel = false
+    const listEntries: string[] = []
+    map.forEachFeatureAtPixel(
+      pixel,
+      (feature) => {
+        if (!(feature instanceof Feature)) {
+          return false
+        }
+        if (!hasFeatureAtPixel) {
+          hasFeatureAtPixel = true
+          overlay.setPosition(map.getCoordinateFromPixel(pixel))
+        }
+        listEntries.push(`<li>${getPrimaryName(feature.get('names'))}</li>`)
+      },
+      {
+        layerFilter: (layer) => layer === vectorLayer,
+      }
+    )
+    if (!hasFeatureAtPixel) {
+      overlay.setPosition(undefined)
+    } else {
+      unregister?.()
```

Comment:
```suggestion
      if (unregister) {
        unregister()
      }
```
Seems more readable in this case